src/lib/hooks/routing/useClientSideSmartOrderRouterTrade.ts

```diff
@@ -78,26 +79,39 @@ export default function useClientSideSmartOrderRouterTrade<TTradeType extends Tr
       return { error: true }
     }
   }, [config, params, queryArgs, wrapType])
+
```

Comment:
nit: why all these new newlines?

Reply:
I inentionally made this one because i thought it made things more readable